docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+- **Pack as a .NET tool**
+
+  When enabled, packs the project as a special package that contains a console application that may be installed via the "dotnet tool" command. This is disabled by default.
+
+- **Package Output Path**
+
+  Determines the output path in which the package will be dropped.
+
+  Defaults to the MSBuild value of `$(OutputPath)`.
+
+- **Assembly neutral language**
+
+  Which language code is considered the neutral language. Defaults to unset.
+
+- **Assembly version**
+
+  The version of the assembly, defaults to `1.0.0.0` if not set.
+
+- **File version**
+
+  The version associated with the file, defaults to `1.0.0.0` if not set.
+
+### License
+
+- **Package License**
+
+  Specify a license fo the project's package. Defaults to `None`.
+
+- **Symbols**
+
+  - **Produce a symbol package**
+
+    When enabled, creates an additional symbol package when the project is packaged. This is disabled by default.
+
+## Code Analysis
+
+Settings related to code analysis.
+
+### All analyzers
+
+Settings related to when analysis runs.
+
+- **Run on build**
+
+  When enabled, runs code analysis on build. Defaults to enabled.
+
+- **Run on live analysis**
+
+  When enabled, runs code analysis live in the editor as you type. Defaults to enabled.
+
+### .NET analysis
+
+Settings related to .NET analyzers.
+
+- **Enforce code style on build (experimental)**
+
+  When enabled, produces diagnostics about code style on build. This is disabled by default.
+
+- **Enable .NET analyzers**
+
+  When enabled, runs .NET analyzers to help with API usage. Defaults to enabled.
+
+- **Analysis level**
+
+  The set of analyzers that should be run in the project. Defaults to `Latest`. For more information, see [MSBuild: AnalysisLevel](/dotnet/core/project-sdk/msbuild-props#analysislevel).
+
+## MAUI Shared
+
+These are project settings for .NET MAUI that are shared across all target platforms.
+
+### General
+
+General settings related to .NET MAUI.
+
+- **Application Title**
+
+  The display name of the application.
+
+- **Application ID**
+
+  The identifier of the application in reverse domain name format, for example: `com.microsoft.maui`.
+
+- **Application ID (GUID)**
+
+  The identifier of the application GUID format.
+
+- **Application Display Version**
+
+  The version of the application. This should be a single digit integer. Defaults to `1`.
+
+## Android
+
+These are Android-specific .NET MAUI settings.
+
+### Manifest
+
+Settings related to the Android manifest.
+
+- **Application name**
+
+  The string that's displayed as the name of the application. This is the name that's shown in the app's title bar. If not set, the label of the app's MainActivity is used as the application name. The default setting is `@string/app_name` refers to the string resource `app_name` location in `Resources/values/Strings.xaml`.
```

Comment:
The default setting is XXXX, which refers to the string resource YYYYY located in ZZZZZZ.?

Reply:
```suggestion
  The string that's displayed as the name of the application. This is the name that's shown in the app's title bar. If not set, the label of the app's MainActivity is used as the application name. The default setting is `@string/app_name`, which refers to the string resource `app_name` location in `Resources/values/Strings.xaml`.
```